Tests/FluentAssertions.Specs/Specialized/AssemblyAssertionSpecs.cs

```diff
@@ -260,6 +260,105 @@ public void When_an_assembly_is_null_and_Should_BeNull_is_asserted_it_should_suc
             act.Should().NotThrow();
         }
     }
+
+    public class BeUnsigned
+    {
+        [Fact]
+        public void Guards_for_unsigned_assembly()
```

Comment:
Let's extend this test to cover both null and empty keys to match what we check for inside `BeUnsigned`.

Reply:
Augment with
```cs
[Theory]
[InlineData(null)]
[InlineData("")]
```

And then we can make the `publicKey` parameter to `FindAssembly.Stub` mandatory